docs/validators/sigstore_cosign.md

```diff
@@ -144,9 +144,22 @@ The secret can for example be created directly from your local `config.json` (fo
 ```bash
 kubectl create secret generic my-secret \
   --from-file=.dockerconfigjson=path/to/config.json \
-  --type=kubernetes.io/dockerconfigjson -n connaisseur
+  -n connaisseur
 ```
 
+The secret can also be generated directly from supplied credentials (which may differ from your local `config.json`, using:
+
+```bash
+kubectl create secret docker-registry my-secret \
+  --docker-server=https://index.docker.io/v1/ \
+  --docker-username='<your username>' \
+  --docker-password='<your password>' \
+  -n connaisseur
+```
+
+!!! note
```

Comment:
I think, we should stick to a common form here as used for example here: https://github.com/sse-secure-systems/connaisseur/blob/master/docs/basics.md#admission-control-validators-and-image-policy
This renders as: https://sse-secure-systems.github.io/connaisseur/v2.5.1/basics/#admission-control-validators-and-image-policy

Reply:
Hey @xopham! No worries, I updated to match the existing syntax. The reason I went for the admonition-style syntax, is because I noted that the admonitions plugin is installed, at https://github.com/sse-secure-systems/connaisseur/blob/master/mkdocs.yml#L36 - these would render as described at https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Anyway, consistency is better than flashiness, so I've reverted to the existing syntax :)

D